config/server.js

```diff
@@ -40,7 +40,8 @@ module.exports = {
     fromAddress: "vam@test.com"
   },
   // key to sign tokens
-  jwtSecretKey: "-",
+  jwtSecretKey:
+    "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzY29wZXMiOlsiYWRtaW4iLCJkYXNoYm9hcmQiLCJyZWFkOmN1c3RvbWVyX2dyb3VwcyIsInJlYWQ6Y3VzdG9tZXJzIiwicmVhZDpmaWxlcyIsInJlYWQ6b3JkZXJfc3RhdHVzZXMiLCJyZWFkOm9yZGVycyIsInJlYWQ6cGFnZXMiLCJyZWFkOnBheW1lbnRfbWV0aG9kcyIsInJlYWQ6cHJvZHVjdF9jYXRlZ29yaWVzIiwicmVhZDpwcm9kdWN0cyIsInJlYWQ6c2V0dGluZ3MiLCJyZWFkOnNoaXBwaW5nX21ldGhvZHMiLCJyZWFkOnNpdGVtYXAiLCJyZWFkOnRoZW1lIiwid3JpdGU6Y3VzdG9tZXJfZ3JvdXBzIiwid3JpdGU6Y3VzdG9tZXJzIiwid3JpdGU6ZmlsZXMiLCJ3cml0ZTpvcmRlcl9zdGF0dXNlcyIsIndyaXRlOm9yZGVycyIsIndyaXRlOnBhZ2VzIiwid3JpdGU6cGF5bWVudF9tZXRob2RzIiwid3JpdGU6cHJvZHVjdF9jYXRlZ29yaWVzIiwid3JpdGU6cHJvZHVjdHMiLCJ3cml0ZTpzZXR0aW5ncyIsIndyaXRlOnNoaXBwaW5nX21ldGhvZHMiLCJ3cml0ZTp0aGVtZSJdLCJqdGkiOiI1YmYzNDVjM2U4YmY4YjRkZjVmNjI4ZGEiLCJlbWFpbCI6ImxvcnRzY2hpQGdtYWlsLmNvbSIsImlhdCI6MTU0MjY2OTc2MywiZXhwIjoxNTQyNzU2MTYzfQ.qyJM6N7keEiYOKutnCZsQ26jaBfkUcnK-L9kHZetcbg",
```

Comment:
I think adding this from a different file as default setting and also adding options for environment variables and user own setting would be best to not to clutter the settings file, or reduce length.

Reply:
- It throws silent error of "const" assign. The variables with "const" changed continuously, so it had to be "let" to get work the code.
- Sure the best solution would be set the jwtSecretKey in the settings individual to admin. But that would be a feature. Feel free to open for it a ticket. I will change it back to a '-' sign again. But it should be mentioned in the docs to know everyone it is an unsafe sign to certify a jwtToken.